src/Blueprints/BlueprintRepository.php

```diff
@@ -3,6 +3,7 @@
 namespace Daynnnnn\StatamicDatabase\Blueprints;
 
 use Daynnnnn\StatamicDatabase\Blueprints\Traits\ExistsAsModel;
+use Daynnnnn\StatamicDatabase\Exceptions\DefaultBlueprintNotFoundException;
```

Comment:
I don't think this class is being defined anywhere?

Reply:
oh... yes, you're right, just forget to commit the file. Fixed :)